hugo/content/5_deep-dives/53_Networking-and-communication/531_ROS-network.md

```diff
@@ -77,4 +77,27 @@ Routed networks can be deployed to a device with the following parameters:
 * **Restart policy**: Kindly refer to the restart policy.
 On reboot, devices configured using DHCP may boot up with a new IP address and the network configuration of a deployed routed network becomes invalid. This can be avoided by assigning a static IP to the device you intend to deploy a routed network to esp in production systems.
 
+## ROS2 Routed Networks Internal
 
+ROS2 Routed Network internally uses eProsima DDS Router for communication.  DDS Router runs a FastDDS discovery server for peer-to-peer communication between cloud deployments and routers.  The router runs in repeater mode so that multiple devices can communicate with each other from one local network to another local network at a different geographic location.
+
+**ServerID** - The DDS Router runs discovery server on the Server ID. By default, the ServerID is set to 0.
+**ServerPort** - The DDS Router runs discovery server on the Server Port. By default, ServerPort is set to 11811.
+
+**DomainID** - DDS Router Client uses the DomainID on the device components to ensure it only listens to the ros2 nodes running on this DomainID. By default, the DomainID is set to 0. We may explicitly set environment variable `ROS_DOMAIN_ID=0` in the user executables to make sure it runs on domain id 0.
```

Comment:
**DomainID** - DDS Router Client uses the DomainID on the device components to ensure it only listens to the ros2 nodes running on this DomainID. This also sets environment variable `ROS_DOMAIN_ID` in the user executables. We may override it by explicitly set environment variable `ROS_DOMAIN_ID=0` in the user executables to make sure it runs on domain id 0.